subgraphs/compound-v3/schema.graphql

```diff
@@ -1103,9 +1103,6 @@ type MarketDailySnapshot @entity {
   " Current TVL (Total Value Locked) of this market "
   totalValueLockedUSD: BigDecimal!
 
-  " Total amount of reserves in this market (in USD) "
-  totalReservesUSD: BigDecimal
```

Comment:
Is this intended?

Reply:
yes, I accidentally forgot to remove from the last PR :(